phoenix-core/src/it/java/org/apache/phoenix/end2end/BaseTenantSpecificViewIndexIT.java

```diff
@@ -148,10 +148,10 @@ private void createAndVerifyIndex(Connection conn, String viewName, String table
                             + "CLIENT MERGE SORT", QueryUtil.getExplainPlan(rs));
         } else {
             String expected = saltBuckets == null ? 
-                    "CLIENT PARALLEL 1-WAY RANGE SCAN OVER _IDX_" + tableName + " [" + Long.toString(Long.MIN_VALUE + expectedIndexIdOffset) + ",'" + tenantId + "','" + valuePrefix + "v2-1']\n"
+                    "CLIENT PARALLEL 1-WAY RANGE SCAN OVER _IDX_" + tableName + " [" + Long.toString(Short.MIN_VALUE + expectedIndexIdOffset) + ",'" + tenantId + "','" + valuePrefix + "v2-1']\n"
```

Comment:
nit: Use `Short.toString` instead.

Reply:
I mostly restore this part from before PHOENIX-3547.
Realizing now it was all over the place. Using  + Short.MIN_VALUE is some place, -32768 in others, toString in yet others.
Here it does not matter though, the output is identical.

Reply:
In fact this is best expressed as "string" + (number1 + number2) + "string", anyway. I'll make that change.

Reply:
Sounds good, thanks